packages/participants-svc/src/application/routes.ts

```diff
@@ -1166,8 +1175,19 @@ export class ExpressRoutes {
             "Received request to check and create liquidity adjustment."
         );
         try {
-            const ignoreDuplicate = req.query.ignoreDuplicate as unknown as boolean ||
-                req.query.ignoreduplicate as unknown as boolean || false;
+            const ignoreDuplicateParam = req.query.ignoreDuplicate as string ||
+                req.query.ignoreduplicate as string || "false";
+            let ignoreDuplicate;
+            if (ignoreDuplicateParam === "false" ||
+                ignoreDuplicateParam === "0" ||
+                ignoreDuplicateParam === "true" || 
+                ignoreDuplicateParam === "1") {
```

Comment:
This can't be done like this, Boolean("false") and Boolean("0") are both true.
```
const ignoreDuplicate = (req.query.ignoreDuplicate !== undefined && (req.query.ignoreDuplicate as string).toUpperCase() === "TRUE");
```